Unix/tests/cli/test_cli.cpp

```diff
@@ -2149,4 +2175,292 @@ NitsTestWithSetup(TestOMICLI32_UserOption, TestCliSetup)
 }
 NitsEndTest
 
+
+NitsTestWithSetup(TestOMICLI36_GetInstanceWsmanKerberosAuth, TestCliSetupSudo)
+{
+    if (runKrbTests && startServer && !travisCI)
+    {
+        NitsDisableFaultSim;
+
+        string out;
+        string err;
+        MI_Char buffer[1024];
+
+        Stprintf(buffer, MI_COUNT(buffer),
+                 MI_T("omicli gi --hostname %T --auth Kerberos -u %T@%T -p %T --port %T oop/requestor/test/cpp { MSFT_President Key 1 }"),
```

Comment:
we only use %T@%T here and others using %T\\%T, looks inconsitent even it is a unittest.
I don't know which one is correct.